include/engine/datafacade/shared_datafacade.hpp

```diff
@@ -227,15 +228,15 @@ template <class EdgeDataT> class SharedDataFacade final : public BaseDataFacade<
 
     SharedDataFacade()
     {
-        if (!datastore::SharedMemory::RegionExists(CURRENT_REGIONS))
+        if (!storage::SharedMemory::RegionExists(storage::CURRENT_REGIONS))
         {
             throw util::exception("No shared memory blocks found, have you forgotten to run osrm-datastore?");
         }
-        data_timestamp_ptr = (SharedDataTimestamp *)datastore::SharedMemoryFactory::Get(
-                                 CURRENT_REGIONS, sizeof(SharedDataTimestamp), false, false)
+        data_timestamp_ptr = (storage::SharedDataTimestamp *)storage::makeSharedMemory(
+                                 storage::CURRENT_REGIONS, sizeof(storage::SharedDataTimestamp), false, false)
```

Comment:
There is a ast node for C-style casts (-> cncc pattern) if we want to detect all of them (hard to grep).